<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreerNouveauMdp</class>
 <widget class="QWidget" name="CreerNouveauMdp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;background-color:white</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="global">
     <item>
      <widget class="QLabel" name="label_erreur">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Informations générales</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="global_general">
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Mot de passe (répéter):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="edit_mdp_0">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="edit_mdp_1">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Mot de passe:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Nom de l'appareil:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Login:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>IP de l'appareil:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="bouton_auto">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Auto</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="edit_login"/>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="edit_ip"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="edit_nom"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>informations utiles</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>931</width>
           <height>651</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="global_supplementaire">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Adresse complette de connexion à l'appareil</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPlainTextEdit" name="edit_autres_informations"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Autres informations utiles:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="edit_address"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Clef de cryptage associee</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="bouton_clef">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Choisir une clef</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Options</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>921</width>
           <height>591</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="global_options">
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Firefox</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>SSH</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Chrome</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/ssh.ico</normaloff>Images/ssh.ico</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Internet Explorer</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/internet_explorer.png</normaloff>Images/internet_explorer.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/firefox.jpg</normaloff>Images/firefox.jpg</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Telnet</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/telnet.jpeg</normaloff>Images/telnet.jpeg</iconset>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>DeviceManager</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/chrome.png</normaloff>Images/chrome.png</iconset>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>Putty</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>DM</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/putty.png</normaloff>Images/putty.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Connexion possible avec ce materiel:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_valider">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_nom</tabstop>
  <tabstop>edit_ip</tabstop>
  <tabstop>edit_login</tabstop>
  <tabstop>edit_mdp_0</tabstop>
  <tabstop>edit_mdp_1</tabstop>
  <tabstop>edit_address</tabstop>
  <tabstop>bouton_clef</tabstop>
  <tabstop>edit_autres_informations</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>bouton_valider</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
